prisma/schema.prisma

```diff
@@ -8,35 +8,62 @@ datasource db {
 }
 
 model ConnectInstallation {
-  id           Int    @id @default(autoincrement())
-  key          String
-  clientKey    String @unique
-  sharedSecret String
-  baseUrl      String
-  displayUrl   String
+  id               Int                @id @default(autoincrement())
+  key              String
+  clientKey        String             @unique @map("client_key")
+  sharedSecret     String             @map("shared_secret")
+  baseUrl          String             @map("base_url")
+  displayUrl       String             @map("display_url")
+  FigmaWebhook     FigmaTeam[]
+  AssociatedDesign AssociatedDesign[]
+
+  @@map("connect_installation")
+}
+
+model AssociatedDesign {
+  id          Int     @id @default(autoincrement())
+  fileKey     String  @map("file_key")
+  nodeId      String  @map("node_id")
+  isPrototype Boolean @map("is_prototype")
+
+  connectInstallation   ConnectInstallation @relation(fields: [connectInstallationId], references: [id], onDelete: Cascade)
+  connectInstallationId Int                 @map("connect_installation_id")
+
+  @@unique([fileKey, nodeId, isPrototype, connectInstallationId])
```

Comment:
@jvrankul23 so are we going to say that two files are different if one is a prototype and one isn't? If that's not the intention, should we make this to instead be
```typescript
@@unique([fileKey, nodeId, connectInstallationId])
```

Reply:
@damoster From what I've been able to gather through testing, files cannot be prototypes (neither can nodes), a prototype is just a specific view in Figma files (similar to inspect mode vs. design mode). @stannnous and I were discussing this the other day and we may end up removing this `isPrototype` field and just storing prototypes as nodes, depending on how discussions with Figma go.

Reply:
Thanks @jvrankul23, in that case I'll leave this as is while we're still storing `isPrototype`. Should hopefully be pretty easy to remove the column down the line if we need to.